### PR TITLE
Sfint-4123 Main Title Component

### PR DESCRIPTION
**Pull request by mmitiche**
# mainTitle 

## The `mainTitle` component is a title to mark the beginning of a screen on the case assist flow.

mainTitle

### CODE REVIEW

File: src/main/translations/fr.translation-meta.xml [hunk 1]
```diff
@@ -74,4 +74,8 @@
     <label>Abandonner la demande</label>
     <name>cookbook_AbandonRequest</name>
   </customLabels>
+  <customLabels>
+    <label>Bonjour, en quoi avez-vous besoin d'aide ?</label>
```
Comment (lbergeron) [inline src/main/translations/fr.translation-meta.xml:78]:
```suggestion
    <label>Bonjour, en quoi avez-vous besoin d'aide?</label>
```

I would suggest this translation.

```suggestion
    <label>Bonjour, comment puis-je vous aider?</label>
```

File: src/main/default/lwc/mainTitle/__tests__/mainTitle.test.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    const expectedTitle = 'Expected Title';
+    element.title = expectedTitle;
+
+    await allPromisesResolution();
+    const titleNode = element.shadowRoot.querySelector(
+      'h1.slds-text-heading_large'
+    );
+    expect(titleNode).not.toBeNull();
+    expect(titleNode.textContent).toBe(expectedTitle);
+    expect(element).toBeAccessible();
```
Comment (lbergeron) [inline src/main/default/lwc/mainTitle/__tests__/mainTitle.test.js:46]: According to `sa11y-jest` documentation, you must `await` when invoking `toBeAccessible`.

**Review by lbergeron — APPROVED** [review]
Please make sure that accessibility issues are well reported when running the tests. The rest looks good to me!

File: src/main/default/labels/CustomLabels.labels-meta.xml [hunk 3]
```diff
@@ -142,4 +142,11 @@
     <protected>false</protected>
     <shortDescription>Abandon Request</shortDescription>
   </labels>
+  <labels>
+    <fullName>cookbook_MainTitle</fullName>
+    <value>Hi, what do you need help with?</value>
```
Comment (mmitiche) [inline src/main/default/labels/CustomLabels.labels-meta.xml:147]: @acarpentier01  please tell me if this label and the french version of it looks fine to you.

Comment (acarpentier01) [reply src/main/default/labels/CustomLabels.labels-meta.xml:147]: @mmitiche The current labels are fine but not a word-for-word match. Here are a couple of options:
1. Hi, how may we help you? Bonjour, comment pouvons-nous vous aider?
2. Hi, need help? Bonjour, besoin d'assistance?

Comment (mmitiche) [reply src/main/default/labels/CustomLabels.labels-meta.xml:147]: Thank you 👌

File: src/main/default/lwc/mainTitle/__tests__/mainTitle.test.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  });
+
+  it('should display the correct value of the title', async () => {
+    const element = createTestComponent();
+    const expectedTitle = 'Expected Title';
+    element.title = expectedTitle;
+
+    await allPromisesResolution();
+    const titleNode = element.shadowRoot.querySelector(
+      'h1.slds-text-heading_large'
```
Comment (erocheleau) [inline src/main/default/lwc/mainTitle/__tests__/mainTitle.test.js:42]: You can put these selectors strings as constants at the top of your test perhaps? You are re-using the same more than once.

**Review by erocheleau — APPROVED** [review]

**Pull request merged**